### PR TITLE
Add protobuf

### PR DESCRIPTION
**Pull request by drasko**
Signed-off-by: Drasko DRASKOVIC <drasko.draskovic@gmail.com>

### CODE REVIEW

**Comment by drasko** [comment]
Should fix https://github.com/mainflux/mainflux/issues/163

File: message.proto [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	string  ContentType = 4;
+	bytes   Payload     = 5;
+}
+
+// Message represents a resolved (normalized) raw message.
+message Message {
+	string  Channel     = 1;
+	string  Publisher   = 2;
+	string  Protocol    = 3;
+	string  Name        = 4 [json_name="n"];
```
Comment (mijicd) [inline message.proto]: json_name can be removed.

File: normalizer/normalizer.go [hunk 2]
```diff
@@ -1,11 +1,11 @@
 package normalizer
 
 import (
-	"encoding/json"
 	"fmt"
 
 	"github.com/cisco/senml"
 	"github.com/go-kit/kit/log"
+	"github.com/golang/protobuf/proto"
```
Comment (mijicd) [inline normalizer/normalizer.go:8]: Please add this dependency to dep file.

**Review by mijicd — CHANGES_REQUESTED** [review]

File: message.proto (hunk 1 above)
Comment (drasko) [reply message.proto]: Agreed, as now all messages need to be parsed via Protobuf.

**Comment by mijicd** [comment]
Closed due to CI issues.

**Pull request closed without merging**